x/wasm/client/proposal_handler_test.go

```diff
@@ -158,7 +158,7 @@ func TestGovRestHandlers(t *testing.T) {
 				"admin":       "cosmos100dejzacpanrldpjjwksjm62shqhyss44jf5xz",
 				"code_id":     "1",
 				"label":       "https://example.com/",
-				"init_msg":    "my/builder:tag",
+				"msg":         "my/builder:tag",
```

Comment:
This is a strange and confusing value for a message

Reply:
You are right. I should set this to something different than a builder.